integration/presto/src/main/java/org/apache/carbondata/presto/CarbonVectorBatch.java

```diff
@@ -73,7 +73,7 @@ public static CarbonVectorBatch allocate(StructField[] schema,
   }
 
   private CarbonColumnVectorImpl createDirectStreamReader(int batchSize, DataType dataType,
-      StructField field, Dictionary dictionary, SliceArrayBlock dictionarySliceArrayBlock) {
+      StructField field, Dictionary dictionary, Block dictionarySliceArrayBlock) {
```

Comment:
better to modify the names also for readability and maintainability purpose.  
dictionarySliceArrayBlock --> dictionaryBlock